<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <arg name="bgd_path" default="$(find map_data)/data/"
         doc="Path to map_data data directory."/>
    <arg name="bgd_name" default="bgd_map.png"
         doc="Name of background image."/>
    <arg name="coords_name" default="buchlovice.mapdata"
         doc="Name of mapdata file."/>

    <node name="gui" pkg="osm2qr" type="gui" output="screen">

        <rosparam subst_value="true">
            bgd_path: $(arg bgd_path)
            bgd_name: $(arg bgd_name)
            coords_name: $(arg coords_name)
            save_path: $(find osm2qr)/data/
        </rosparam>
    </node>
</launch>
